File: src/main/Java/boj_step2/boj2525.java
import java.io.*;
import java.util.StringTokenizer;

public class boj2525 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int h = Integer.parseInt(tokenizer.nextToken());
        int m = Integer.parseInt(tokenizer.nextToken());
        int time = Integer.parseInt(reader.readLine());
        int day = 24;
        int hour = 60;
        h += (time + m) / hour;
        m += time % hour;
        h %= day;
        m %= hour;
        writer.append(h + " " + m);
        writer.close();
    }
}

File: src/main/Python/boj_step2/boj2525.py
import sys

h, m = map(int, sys.stdin.readline().split())
time = int(sys.stdin.readline())
h += time // 60
m += time % 60
if m >= 60:
    h += 1
    m %= 60
if h >= 24: h %= 24
print(h, m)